doc/nestml_language/nestml_language_concepts.rst

```diff
@@ -272,16 +272,13 @@ For example, the following model will result in one warning and one error:
 .. code-block:: nestml
 
    neuron test:
```

Comment:
Could you add some text in the very beginning of this file to indicate that NESTML uses whitespace to demarcate blocks?

Reply:
I agree.
Also, it looks like we used two spaces previously and now we're using four. Is four mandatory, or is it as in Python where indentation just has to be consistent? This needs documentation early on.

Reply:
I have updated the documentation now. The indentation is similar to python where the spaces/tabs must be consistent.